recipes/pyrdl/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "pyrdl" %}
+{% set version = "1.1.3_rdkit" %}
```

I'm assuming that the `_rdkit` is not part of the versioning scheme, right?
```suggestion
{% set version = "1.1.3" %}
```